ci/prow/config_start.yaml

```diff
@@ -31,6 +31,13 @@ metadata:
 data:
   config: ""
 ---
+apiVersion: v1
```

Comment:
I think we need a comment here mentioning it's for Boskos configmap

Reply:
Done.